### PR TITLE
Testflight

### PR DESCRIPTION
**Pull request by michaelneale**
This actually deploys to testflight automatically via xcode cloud believe it or not!

### CODE REVIEW

**Review by ethan-tbd — APPROVED** [review]
thx @michaelneale big unblock!

File: ios/ci_scripts/ci_post_clone.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+# Install Flutter using git.
+git clone https://github.com/flutter/flutter.git --depth 1 -b stable $HOME/flutter
+export PATH="$PATH:$HOME/flutter/bin"
+
+# Install Flutter artifacts for iOS (--ios), or macOS (--macos) platforms.
+flutter precache --ios
+
+# Install Flutter dependencies.
+flutter pub get
```
Comment (wesbillman) [inline ios/ci_scripts/ci_post_clone.sh:17]: In the future, we should probably use hermit here. Hopefully, we just need a ```sh
./bin/activate-hermit
just get
```

**Review by wesbillman — APPROVED** [review]

**Comment by wesbillman** [comment]
@michaelneale I rebased this PR with `main` as well.

File: ios/ci_scripts/ci_post_clone.sh (hunk 1 above)
Comment (michaelneale) [reply ios/ci_scripts/ci_post_clone.sh:17]: oh yes might try that on another branch - to get used to to xcode build env

**Pull request merged**